extensions/typescript-language-features/src/languageFeatures/completions.ts

```diff
@@ -63,19 +63,21 @@ class MyCompletionItem extends vscode.CompletionItem {
 	) {
 		super(tsEntry.name, MyCompletionItem.convertKind(tsEntry.kind));
 
-		if (tsEntry.source && tsEntry.hasAction) {
+		if (tsEntry.source && tsEntry.hasAction && client.apiVersion.lt(API.v500)) {
```

Comment:
https://dev.azure.com/vscode/VSCode/_build/results?buildId=87510&view=logs&j=c7493abb-a1f4-533f-2d24-71780a69f247&t=8d02ba2f-f021-560e-034c-effb4dd4b969&l=324

> Property 'v500' does not exist on type 'typeof API'.

Reply:
If you want, I'm pretty sure we've returned the correct `sortText` since TypeScript 4.4.1 (https://github.com/microsoft/TypeScript/issues/45288) - so `v490` would be a reasonable version gate.